ci/tests/puppeteer/scenarios/login-sso-stale-tgc/config/application.properties

```diff
@@ -0,0 +1,3 @@
+# This property is in property file rather than commadn line
+# so test on Windows (with msys2 terminal) doesn't convert it to path (e.g. /cas -> c:\msys64\cas)
+cas.tgc.path='/cas'
```

Comment:
I had to move this property into a property file. The only other way I could think of that might work would be to have something in the SPEL context taht would evaluate to a forward slash, sort of like ${forwardSlash}. The is no escaping that will get around the msys2/cygwin path fixing and turning off the path fixing doesn't work b/c other things rely on it.

Reply:
This is unfortunate, but it's fine. As long tests pass, I am not sure it makes any difference.